File: BOJ_2460/BOJ_2460.c
#include <stdio.h>
int main(){
	int ANS = 0, HOLDER = 0;
	int N, M;
	for(int i=0; i<10; i++){
		scanf("%d %d", &N, &M);
		ANS -= N;
		ANS += M;
		if(ANS>HOLDER) HOLDER = ANS;
	}
	printf("%d\n", HOLDER);
	return 0;
}

File: BOJ_2460/BOJ_2460.java
import java.io.*;
import java.util.*;

class Main{
	public static void main(String[] args)
			throws IOException{
			BufferedReader BR = new BufferedReader(new InputStreamReader(System.in));
			BufferedWriter BW = new BufferedWriter(new OutputStreamWriter(System.out));
			int ANS = 0, HOLDER = 0;
			for(int i=0; i<10; i++){
				StringTokenizer ST = new StringTokenizer(BR.readLine());
				int N = Integer.parseInt(ST.nextToken());
				int M = Integer.parseInt(ST.nextToken());
				ANS -= N;
				ANS += M;
				if(ANS>HOLDER) HOLDER = ANS;

			}
			BW.write(HOLDER + "\n");
			BW.flush();


	}
}
